audplot/core/api.py

```diff
@@ -0,0 +1,178 @@
+import typing
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+
+import audmetric
+
+
+def confusion_matrix(
+        truth: typing.Union[typing.Sequence, pd.Series],
+        prediction: typing.Union[typing.Sequence, pd.Series],
+        *,
+        labels: typing.Sequence = None,
+        percentage: bool = True,
+):
+    r"""Confusion matrix between ground truth vs. predicted labels.
+
+    The confusion matrx is calculated by :mod:`audmetric.confusion_matrix`.
+
+    Args:
+        truth: truth values
+        prediction: predicted values
+        labels: labels to be included in confusion matrix
+        percentage: if ``True`` present the confusion matrix
+            with percentage values instead of absolute numbers
+
+    Example:
+        .. plot::
+            :context: reset
+            :include-source: false
+
+            from audplot import confusion_matrix
```

Comment:
do we need this?

Reply:
Yes, maybe I should add a comment as well.

The problem is when using the `plot::` directive from matplotlib the functions are not automatically available as it would be the case without using `plot::`. So we need to import them. But not in order to spam the examples and make them look the same as in the other packages, I now silently import the functions first.